sdk/communication/Azure.Communication.Calling.Server/src/ConversationClient.cs

```diff
@@ -410,6 +410,66 @@ public virtual Response<GetCallRecordingStateResponse> GetRecordingState(string
             }
         }
 
+        /// <summary> Play Audio. </summary>
+        /// <param name="conversationId"> The conversation id. </param>
+        /// <param name="audioFileUri"> The uri of the audio file. </param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<PlayAudioResponse>> PlayAudioAsync(string conversationId, Uri audioFileUri, string operationContext, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(PlayAudioAsync)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(conversationId, nameof(conversationId));
+                Argument.AssertNotNull(audioFileUri, nameof(audioFileUri));
+
+                PlayAudioRequest request = new PlayAudioRequest()
+                {
+                    AudioFileUri = audioFileUri.AbsoluteUri,
+                    Loop = false,
```

Comment:
Please note, I have set the default value of the Loop=false while making a call to RestApi. This will not give contoso to set the Loop value while using SDK. Please let me know if this is okay for our scenario ?
@bharat-kalyan-namburi @chrwhit @navali-msft ?

Reply:
I remember us talking about that, did we come to a decision?

Reply:
Please add a comment with why you are hard coding it to false.

Reply:
@chrwhit, Sorry I missed your comment, So the decision was to remove the CancelMedia Operation api and Only Add the PlayAudio Api where we do not support loop on media for PlayAudio out-call. I hope this answers your question :)

Reply:
Yes this sounds fine as per our discussion yesterday

Reply:
Great, Thank you for the confirmation Ajay!
Resolving the comment.